inc/ThirdParty/Plugins/I18n/WPML.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace WP_Rocket\ThirdParty\Plugins\I18n;
+
+use WP_Rocket\Event_Management\Subscriber_Interface;
+use WP_Rocket\ThirdParty\ReturnTypesTrait;
+
+/**
+ * Subscriber for compatibility with WPML.
+ */
+class WPML implements Subscriber_Interface {
+	use ReturnTypesTrait;
+
+	/**
+	 * Events for subscriber to listen to.
+	 *
+	 * @return array
+	 */
+	public static function get_subscribed_events() {
+		if ( ! defined( 'ICL_SITEPRESS_VERSION' ) ) {
+			return [];
+		}
+
+		$events = [
+			'wcml_is_cache_enabled_for_switching_currency' => 'return_true',
+			'rocket_rucss_is_home_url'                     => [ 'is_secondary_home', 10, 2 ],
+		];
+
+		return $events;
+	}
+
+	/**
+	 * Checks if page to be processed is secondary homepage.
+	 *
+	 * @param string $home_url home url.
+	 * @param string $url url of current page.
+	 * @return string
+	 */
+	public function is_secondary_home( string $home_url, string $url ): string {
+		global $sitepress;
+
+		// Get active languages on site.
+		$languages = $sitepress->get_active_languages();
```

Comment:
can the `get_active_languages()` method return something other than an array?

Reply:
@Tabrisrp from what I see on the WPML core, it will return only an array. like something below:

```
Array
(
 [en] => Array
  (
   [id] => 1
   [active] => 1
   [default_locale] => en_US
   [native_name] => English
   [missing] => 0
   [translated_name] => English
   [language_code] => en
   [country_flag_url] => http://yourdomain/wpmlpath/res/flags/en.png
   [url] => http://yourdomain/about
  )

 [fr] => Array
  (
   [id] => 4
   [active] => 0
   [default_locale] => fr_FR
   [native_name] => Français
   [missing] => 0
   [translated_name] => French
   [language_code] => fr
   [country_flag_url] => http://yourdomain/wpmlpath/res/flags/fr.png
   [url] => http://yourdomain/fr/a-propos
  )

 [it] => Array
  (
   [id] => 27
   [active] => 0
   [default_locale] => it_IT
   [native_name] => Italiano
   [missing] => 0
   [translated_name] => Italian
   [language_code] => it
   [country_flag_url] => http://yourdomain/wpmlpath/res/flags/it.png
   [url] => http://yourdomain/it/circa
  )
)
```